### DISCUSSION TITLE
Not all properties are useable / visible in Filter Control

### DISCUSSION THREAD

**Discussion by waddock** [microsoft-search/pnp-modern-search#2028] [General]
Hi there.

I have an issue where I am not able to use certain values from indexed items within the "Search Filters". 

Properties such as "FileType" and "ContentType" work ok, however my custom columns with either "Text" or "Lookup" do not appear in the filter web part.

The steps I have taken to configure are:

- Create a "Custom Site Column" - Called "Ben_Test" with a type of "Text"
- Added "Ben_Test" Custom Site Column to a document library
- Added content to document library and added values for "Ben_Test"
- Added my account to "Site Collection Administrators" (Prevents bug with Search Schema Admin UX not displaying crawled property)
- Mapped RefinableString01 to "ows_Ben_Test" in SharePoint search Administration
- Reindexed the site with the document library
- Added the Ben_Test property to the PnP Search Results - This shows as "BenTestOWSTEXT"
- Tested PnP Search Results in Debug mode to ensure the value shows in results (It works)
- Tested native SharePoint search using a value held within "ows_Ben_Test" (This works - proves the value is indexed)
- Tested PnP Search Box using value held within "ows_Ben_Test" (This works - proves the value is within the search results)
- Configured PnP Search Filter to filter Fields "FileType", "ContentType" & "BenTestOWSTEXT" with Combo Box
- Published the page.

All I get in the filter box is the values for "FileType" & "ContentType" yet I know the results have values "BenTestOWSText".

What am I doing wrong? 

I have followed each step within these articles to get this far:

https://docs.microsoft.com/en-gb/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search

https://sympmarc.com/2021/04/13/crawled-properties-not-created-from-site-columns-in-modern-team-sites/

Thanks in advance for any assistance you can provide.
Ben

**Comment by wobba** [comment]
You can only use refinable managed properties as a filter. `BenTestOWSTEXT` is not refinable. We will likely change the UX and hardcode all known refinable properties to avoid people making mistakes (multiple issues logged on this). Use e.g. `RefinableString01`and you should be fine.
